src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -325,14 +325,35 @@ std::string ContextAttribute::renderAsJsonObject
   {
     if (omitValue == false)
     {
+      std::string effectiveValue = "";
+      bool valueIsNumberOrBool = false;
+      switch(valueType)
+      {
+      case ValueTypeString:
+        effectiveValue = stringValue;
+        break;
+      case ValueTypeBoolean:
```

Comment:
An empty line between 'break' and 'switch' makes the whole thing a lot easier to read